api/src/pcapi/routes/backoffice_v3/templates/components/search/result_card.html

```diff
@@ -1,9 +1,7 @@
 <div class="card shadow">
     <div class="card-body">
         <h5 class="card-title mb-3">
-            <span class="badge rounded-pill text-bg-primary">
-                {{ row.isActive | format_state }}
-            </span>
+
```

Comment:
D'après le ticket, « Seulement l'étiquette “Suspendu” doit être affichée lorsque isActive=false ».
Donc il faut a priori garder ce bloc, mais le conditionner à `{% if row.isActive %}`

Reply:
`{% if not row.isActive %}` plutôt non ?